src/app/pages/signing/signing.component.html

```diff
@@ -176,23 +185,124 @@ <h5>Emojis</h5>
+            'embed link.'
+            If you don't
+            do this step, your link may not display your photo in your message properly. For example,
+            <a target="_blank" href="https://bit.ly/2NRCMvu">this link</a> will not
+            work but <a target="_blank" href="https://bit.ly/2BuNxkA">this one</a> will (and you can get from one to the
+            other by using the 'Open Image in New Tab' technique just mentioned!).
+          </li>
+          <li>If you are not able to do the 'Open Image in New Tab' method for some reason (e.g. notably Google Drive
+            doesn't let you do this), then you will have to research 'how to get the embed link' for whatever service
+            (e.g. Google Drive) that you are trying to get the embed link from.
+          </li>
+        </ol>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- Embed Photo Modal -->
+<div class="modal fade" id="embed-photo" tabindex="-1" role="dialog" aria-labelledby="embed-photo-label"
+     aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered modal-lg" id="embed-photo-dialog" role="document">
     <div class="modal-content">
       <div class="modal-header">
-        <h4 class="modal-title" id="experience-title">Do I need a larger screen to use Virtrolio?</h4>
+        <h5 class="embed-photo-title">Say cheese! Let's add an image!</h5>
         <button type="button" class="close" data-dismiss="modal" aria-label="Close">
           <span aria-hidden="true">&times;</span>
         </button>
       </div>
+      <div class="modal-body d-md-flex p-4" id="embed-photo-div">
+        <div class="col-md-6 my-3 px-lg-5 px-3 vertical-center">
+          <div class="mb-3">
+            <span class="mr-3"><strong>Add a link to a photo!</strong></span>
+            <button class="btn-sm btn-primary" data-toggle="modal" data-target="#embed-modal">
+              Need help?
+            </button>
+          </div>
+          <div class="input-group">
+            <label for="embed-link-field" style="display:none">Insert embed link</label>
+            <input class="form-control" type="text" placeholder="Put your link here!"
+                   [(ngModel)]='embedLink' id="embed-link-field">
+          </div>
+          <div class="mt-3">
+            <label for="width-slider-field" style="display: none">Image Size (%)</label>
+            <label for="width-slider" class="mx-2">Image Size (%):</label>
+            <input [(ngModel)]="imageWidth" type="number" id="width-slider-field" min="0" max="100"
+                   (keyup)="checkSliderValue()">
+            <input [(ngModel)]="imageWidth" class="mx-2" type="range" id="width-slider" min="0" max="100">
+            <button class="btn btn-sm btn-info" (click)="imageWidth=50">Reset</button>
```

Comment:
Reset button should either be next to the text box, or add more margin above it - it's too close to the text box on mobile
![image](https://user-images.githubusercontent.com/39015447/87462160-7e8f8280-c5dd-11ea-8f28-486036be104d.png)